src/mpi/coll/iallgatherv/iallgatherv_tsp_recexch.c

```diff
@@ -122,8 +122,9 @@ int MPIR_TSP_Iallgatherv_sched_intra_recexch_step2(int step1_sendto, int step2_n
 {
     int mpi_errno = MPI_SUCCESS;
     int mpi_errno_ret ATTRIBUTE((unused)) = MPI_SUCCESS;
-    int phase, i, j, count, nbr, offset, rank_for_offset;
-    int x, vtx_id;
+    int phase, i, j, nbr, rank_for_offset;
+    MPI_Aint x;
+    int count, offset, vtx_id;
```

Comment:
Does `count, offset` need be in `MPI_Aint`?

Reply:
same